app/src/panels/label/panel-label.vue

```diff
@@ -22,24 +105,41 @@ export default defineComponent({
 </script>
 
 <template>
-	<div class="label type-title selectable" :class="{ 'has-header': showHeader }" :style="{ color: color }">
-		{{ text }}
+	<div
+		ref="labelContainer"
+		class="label type-title selectable"
+		:class="[font, { 'has-header': showHeader }]"
+		:style="{ color: color, alignItems: showHeader ? 'flex-start' : 'center' }"
+	>
+		<p
+			ref="labelText"
+			:style="{ whiteSpace, fontWeight, textAlign, fontStyle, fontSize: fontSize !== 'auto' ? fontSize : undefined }"
+		>
+			{{ text }}
+		</p>
 	</div>
 </template>
 
 <style scoped>
```

Comment:
```
▲ [WARNING] CSS nesting syntax is not supported in the configured target environment ("chrome87", "edge88", "es2020", "firefox78", "safari14") [unsupported-css-nesting]

    <stdin>:9595:0:
      9595 │ &.sans-serif[data-v-073cd843] {
           ╵ ^
```

Unfortunately, not quite there 😢: https://caniuse.com/css-nesting

```suggestion
<style lang="scss" scoped>
```